lms/djangoapps/instructor_task/api.py

```diff
@@ -471,6 +472,18 @@ def submit_export_ora2_data(request, course_key):
     return submit_task(request, task_type, task_class, course_key, task_input, task_key)
 
 
+def submit_students_grades_data(request, course_key):
+    """
+    AlreadyRunningError is raised if an students grades report is already being generated.
```

Comment:
Seems title and body of the docstring missing. Also, nice to add some pydoc markup for exception description.

Reply:
I used a template solution. And, I think, it is not necessary for this function.